pkg/healthcheck/healthcheck.go

```diff
@@ -0,0 +1,30 @@
+// healthcheck package provides service to get grpc server health status.
```

Comment:
Would you mind adding the word `Package` before `healthcheck`? 

see https://blog.golang.org/godoc

Reply:
Thanks for reviewing. Fixed comment.